app/controllers/members_controller.rb

```diff
@@ -77,5 +77,13 @@ def steal_identity
     redirect_to root_path
   end
 
+  def show_roles
+  end
+
+  def find_roles
+    $roles = Member.find(params[:member_id]).roles
+    $member = Member.find(params[:member_id]).full_name
```

Comment:
$ indicates a global value in ruby, which may interact with other things in unexpected ways
Use "@" syntax instead

Reply:
Also this crashes if you enter an invalid member ID.
Should be surrounded by a try-except block. If it fails, you can flash an error message:
`flash[:error] = t('some.error.translation')`

Reply:
Which is why the variables are set to nil after rendered in the view.
I agree about using begin-rescue though.

Reply:
There is no guarantee that the nil assignment will execute tho

Reply:
Why not?